src/compensations/utils/salary.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  isSalariesType,
+  Salaries,
+} from "studio/components/salariesInput/utils/parseSalaries";
+import { SalariesByLocation } from "../../../studio/lib/payloads/compensations";
+import { Result, ResultError, ResultOk } from "../../../studio/utils/result";
+
+export function calculateSalary(
+  examinationYear: number,
+  degree: string,
+  salaries: Salaries,
+): number | undefined {
+  const degreeValue = degree === "bachelor" ? 1 : 0;
+  const adjustedYear = examinationYear + degreeValue;
+  return salaries[adjustedYear];
+}
+
+export function calculatePension(salary: number): number {
+  return Math.round(salary * 0.07);
```

Comment:
We have a pension field in sanity as well. Maybe we should update this to use that as well

Reply:
Yep, can do that in a separate PR :+1: